prowler/providers/aws/services/cognito/cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in/cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in.py

```diff
@@ -0,0 +1,43 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.cognito.cognito_idp_client import cognito_idp_client
+
+
+class cognito_user_pool_advanced_security_adaptative_authentication_block_sign_in(
+    Check
+):
+    def execute(self):
+        findings = []
+        for pool in cognito_idp_client.user_pools.values():
+            report = Check_Report_AWS(self.metadata())
+            report.region = pool.region
+            report.resource_name = pool.name
+            report.resource_id = pool.id
+            report.resource_arn = pool.arn
+            report.resource_tags = pool.tags
+            if pool.advanced_security_mode == "ENFORCED" and all(
+                [
+                    pool.risk_configuration.account_takeover_risk_configuration.low_action
+                    and pool.risk_configuration.account_takeover_risk_configuration.low_action.get(
```

Comment:
omit the EventAction attribute and add it in the service to the action

Reply:
Done